docs/resources/bms_instance.md

```diff
@@ -0,0 +1,218 @@
+    Text and text files can be injected.
+    Changing this creates a new instance.
+
+* `admin_pass` - (Optional, String, ForceNew) Specifies the administrative password to assign to the instance.
+    Changing this creates a new instance.
+
+* `key_pair` - (Optional, String, ForceNew) Specifies the name of a key pair to put on the instance.
+    The key pair must already be created and associated with the tenant's account.
+    Changing this creates a new instance.
+
+* `security_groups` - (Optional, List, ForceNew) Specifies a array of one or more security group ids to associate
+    with the instance. Changing this creates a new instance.
+
+* `eip_id` - (Optional, String, ForceNew) The ID of the EIP. Changing this parameter will create a new resource.
+
+-> **Note:** If the eip_id parameter is configured, you do not need to configure the bandwidth parameters:
+  `iptype`, `eip_charge_mode`, `bandwidth_size`, `share_type` and `bandwidth_charge_mode`.
+
+* `iptype` - (Optional, String, ForceNew) Elastic IP type. Changing this parameter will create a new resource.
+
+* `eip_charge_mode` - (Optional, String, ForceNew) Elastic IP billing type. Changing this parameter will create a new resource.
+
+* `sharetype` - (Optional, String, ForceNew) Bandwidth sharing type. Changing this parameter will create a new resource.
+
+* `bandwidth_size` - (Optional, Int, ForceNew) Bandwidth size. Changing this parameter will create a new resource.
+
+* `bandwidth_charge_mode` - (Optional, String, ForceNew) Bandwidth billing type. Changing this parameter will create a new resource.
+
+* `system_disk_type` - (Optional, String, ForceNew) Specifies the system disk type of the instance.
+  For details about disk types, see
+  [Disk Types and Disk Performance](https://support.huaweicloud.com/intl/en-us/productdesc-evs/en-us_topic_0014580744.html)
+    Available options are:
+	* `SSD`: ultra-high I/O disk type.
+	* `GPSSD`: general purpose SSD disk type.
+	* `SAS`: high I/O disk type.
+    Changing this creates a new instance.
+
+* `system_disk_size` - (Optional, int, ForceNew) Specifies the system disk size in GB. The value ranges from 40 to 1024.
+    The system disk size must be greater than or equal to the minimum system disk size of the image.
+    Changing this creates a new instance.
+
+* `data_disks` - (Optional, List, ForceNew) Specifies an array of one or more data disks to attach to the instance.
+    The data_disks object structure is documented below. A maximum of 59 disks can be mounted.
+    Changing this creates a new instance.
+
+* `tags` - (Optional, Map, ForceNew) Specifies the key/value pairs to associate with the instance.
+    Changing this creates a new instance.
+
+* `enterprise_project_id` - (Optional, String, ForceNew) Specifies a unique id in UUID format of enterprise project .
+    Changing this creates a new instance.
+
+* `charging_mode` - (Optional, String, ForceNew) Specifies the charging mode of the instance.
```

Comment:
we should update the docs if only *prePaid* is supported.

Reply:
fixed